pep-0573.rst

```diff
@@ -77,6 +75,29 @@ Heap Type
 A type object created at run time.
 
 
+Defining Class
+--------------
+
+The defining class of a method (either bound or unbound) is the class on which
+the method was defined.
+A class that merely inherits the method from its base is not the defining class.
+
+For example, ``int`` is the defining class of ``True.to_bytes``,
+``True.__floor__`` and ``int.__repr__``.
+
+(For methods defined in Python, where unbound methods are plain functions,
+the term is somewhat fuzzy. This PEP deals with methods defined in C
```

Comment:
It isn't normally fuzzy in Python - it's what ends up in the `__class__` closure cell.